glide.yaml

```diff
@@ -65,3 +65,4 @@ import:
   version: v1.1.0
 - package: github.com/onsi/ginkgo
   version: v1.3.1
+- package: github.com/duglin/vlinker
```

Comment:
via what mechanism? just add a "git clone" to the Makefile?  Glide does seem to pull it down ok.